src/show/pages/public/Standings/index.jsx

```diff
@@ -1,70 +1,70 @@
-import { ConstructorStandingCard } from '../../../components';
-import DriverStandingCard from '../../../components/DriverStandingCard';
+import {
+  ConstructorStandingCard,
+  DriverStandingCard
+} from '../../../components';
 
 const Standings = () => {
+  const driver = {
+    name: 'Charles Leclerc',
+    points: '267',
+    place: '1',
+    avatar_url:
+      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
+  };
+
+  const constructor = {
+    constructor_name: 'Ferrari',
```

Comment:
The name of this attribute on the backend, in the Constructor model, should be name, not constructor_name since it already is referring to a constructor